### ISSUE TITLE
'"validateForeignKey " contains extra space' bug still present in XSD 4.1 and at XSD URIs

### ISSUE THREAD

**Issue by leonardoarcari** [liquibase/liquibase#1517]
## Environment
**Liquibase Version**: 4.1.1

**Liquibase Integration & Version**: liquibase-core (Java API)

**Liquibase Extension(s) & Version**: N/A

**Database Vendor & Version**: Oracle Express Edition 11g

**Operating System Type & Version**: Windows 10 (19041)

## Description
Issue described also here [CORE-3534](https://liquibase.jira.com/browse/CORE-3534) and here #1239 

I'm developing a Java application leveraging `liquibase-core` Java API. It fails for any XML changelog script with XSD version >= 3.7 with error:

`Caused by: oracle.xml.parser.schema.XSDException: Invalid value 'validateForeignKey ' for attribute: 'name'`

As far I can see this was partially fixed by #1239, but a regression occurred in versions **4.0** and **4.1**.

Moreover, while I see this fixed in **3.7, 3.8** and **3.9** code, fixed schemas are not available at schema URIs:

* **3.7**: http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.7.xsd
* **3.8**: http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd
* **3.9**: http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd

And of course, 4.0 and 4.1 as well.

https://liquibase.jira.com/browse/CORE-3534

## Steps To Reproduce
Write any changelog XML script with the following `DatabaseChangelog` tag:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<databaseChangeLog
    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
	http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd"/>
```

## Actual Behavior
N/A

## Expected/Desired Behavior
N/A

## Screenshots (if appropriate)
N/A

## Additional Context
N/A

**Comment by molivasdat** [comment]
HI @leonardoarcari Thanks for writing up this issue.  I'll check to see why the fix was not pushed for the other version 3.6-3.9 and thanks for the update on 4.0, and 4.1.

**Comment by molivasdat** [comment]
Hi @leonardoarcari The URLs for 3.7-3.9 have been fixed.


    3.7: http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.7.xsd
    3.8: http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd
    3.9: http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd

The ones for 4.0, 4.1 and 4.2 will be fixed when the 4.2 release comes out and the 4.2 jar file will contain all of the fixed xsds.  In order to use the local ones, you will need to upgrade to 4.2 when it comes out shortly.

**Comment by leonardoarcari** [comment]
@molivasdat Huge! Many thanks guys 👏
